docs/src/pages/docs/crocks/Result.md

```diff
@@ -0,0 +1,1422 @@
+---
+title: "Result"
+description: "Result Crock"
+layout: "guide"
+functions: ["trycatch", "eithertoresult", "firsttoresult", "lasttoresult", "maybetoresult"]
+weight: 130
+---
+
+```haskell
+Result e a = Err e | Ok a
+```
+
+Result is a Sum Type a step above [`Maybe`][maybe]. With a [`Maybe`][maybe] the
+left side represents the lack of a valu, where as with a `Result` the left
+contains
+the error information from an operation. `Result` is well suited for capturing
+disjunction when the cause of the "error" case needs to be communicated. For
+example, when executing a function and you exception is important or useful.
+
+A `Result` represents disjunction by using two constructors, [`Err`](#err) and
+[`Ok`](#ok). An [`Ok`](#ok) instance represents the positive result while
+[`Err`](#err) is considered the negative. With the exception of
+[`coalesce`](#coalesce), [`swap`](#swap) and [`bimap`](#bimap), all `Result`
```

Comment:
One thing that needs to be kept in mind with the limiting to 80 chars is that lines cannot end or begin with a code tick. The space between words cannot be sussed out. So in these descriptions the starts and ends be code ticked text.

Reply:
So what you're saying is if you have something like "[`Ok`](#ok)" then it has to have a word in front of it if it's the beginning of a line?

Reply:
yeppers or a word after it if it is at the end.